### PR TITLE
Improve RecordsChanged error message for debugging

### PR DESCRIPTION
**Pull request by ahorner**

### CODE REVIEW

File: lib/restforce/db/cleaner.rb [hunk 1]
```diff
@@ -13,10 +13,16 @@ class RecordsChanged < RuntimeError
 
         # Public: Initialize a new RecordsChanged exception.
         #
+        # database_model   - A String name of an ActiveRecord class.
         # salesforce_model - A String name of a Salesforce object type.
         # ids              - An Array of String Salesforce IDs.
-        def initialize(salesforce_model, ids)
-          super <<-MESSAGE
+        # timestamps       - A list of Time objects for the lookup.
```
Comment (rdunlop) [inline lib/restforce/db/cleaner.rb]: different name for the parameter than is used in the prototype

**Comment by rdunlop** [comment]
looks good

File: lib/restforce/db/cleaner.rb [hunk 2]
```diff
@@ -80,7 +86,18 @@ def invalid_salesforce_ids
         all_ids = all_salesforce_ids
 
         updated_ids = valid_ids - all_ids
-        DB.logger.error(RecordsChanged.new(@mapping.salesforce_model, updated_ids)) unless updated_ids.empty?
+
+        unless updated_ids.empty?
+          exception = RecordsChanged.new(
+            @mapping.database_model.to_s,
+            @mapping.salesforce_model,
+            updated_ids,
```
Comment (danhodos) [inline lib/restforce/db/cleaner.rb]: Is it potentially useful to log `valid_ids` and `all_ids` here?

Comment (ahorner) [inline lib/restforce/db/cleaner.rb]: Hm, possibly. I'll tweak it a little.

**Comment by ahorner** [comment]
Ready for another peek.

**Comment by rdunlop** [comment]
:+1:

**Pull request merged**